src/components/Event.js

```diff
@@ -49,31 +49,31 @@ const RSVP = () => {
             height="480"
             title="travel"
           ></iframe>
-        </RSVPMap>
-      </RSVPDetails>
-      <RSVPImage>
+        </EventMap>
+      </EventDetails>
+      <EventImage>
         <img
           src="https://images.unsplash.com/photo-1535185384036-28bbc8035f28?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=668&q=80"
           alt="wedding"
         />
-      </RSVPImage>
-    </RSVPContainer>
+      </EventImage>
+    </EventContainer>
   );
 };
 
-export default RSVP;
+export default Event;
```

Comment:
🤔 Can the export go on line 6?

```
export default Event = ...
```

Reply:
Yes, can absolutely move export to line 6. VSCode suggests that it would have to be a functional component, as in it will need to be 

`export default function Event ...`

Is it because it's better for team settings to understand if an component is indeed exported right at the top?

Reply:
Fixed!